browsers/opera_android.json

```diff
@@ -480,9 +480,15 @@
         "83": {
           "release_date": "2024-06-25",
           "release_notes": "https://blogs.opera.com/mobile/2024/06/opera-android-version-image-generation/",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "126"
+        },
+        "84": {
+          "release_date": "2024-08-26",
+          "status": "current",
+          "engine": "Blink",
+          "engine_version": "128"
```

Comment:
In the release note that you mentioned, you can see that Chromium 127 is used here.

https://forums.opera.com/topic/73545/opera-for-android-84

Reply:
Opened https://github.com/mdn/browser-compat-data/pull/24424